Split terms.html

odscjames opened the pull request.

ghost commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Set up Hotjar](https://trello.com/c/KLHW48Fc/18-set-up-hotjar)

jarofgreen committed:
Split terms.html into many templates using includes

(This make it possible for instances of COVE to add extra trackers)

jarofgreen committed:
Add Terms for Google Analytics, behind a config switch

The actual tracking code for Google Analytics is in this library (behind a config switch) so the terms and conditions should be too.

jarofgreen committed:
Release v0.11.0

Bjwebb commented on cove/templates/terms.html:
```diff
@@ -1,217 +1,30 @@
+            {# The content is split into many small templates.
+               This is so if one cove instance wants to change only one small bit of the page, they can do so by providing replacement templates of only the bit they want to change.
+               For example, a cove instance may want to keep most of the page the same but add legal content for another tracker.
+            #}
```
Note these comment tags don't work in Django, instead use https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#comment

odscjames replied:
Don't they? I did test it, wierd .... ok.

odscjames replied:
fixed and tested again

Bjwebb replied:
Weird. Are you testing with a later Django version?
